pkg/okteto/context.go

```diff
@@ -73,6 +73,7 @@ type OktetoContext struct {
 	Analytics         bool                 `json:"-" yaml:"-"`
 	ClusterType       string               `json:"-" yaml:"-"`
 	IsOkteto          bool                 `json:"isOkteto,omitempty" yaml:"isOkteto,omitempty"`
+	IsInsecure        bool                 `json:"isInsecure,omitempty" yaml:"isInsecure,omitempty"`
```

Comment:
Do we need the tags for json and yaml?

Reply:
I mean, having them could imply that those fields are printed/stored in a file while marshaling to json or yaml

Reply:
`Analytics` doesn't have flags for yaml and it can be read [here](https://github.com/okteto/okteto/blob/56df083df7af89b6373ba812bfc4a3519732af7c/pkg/analytics/track.go#L455).

Be aware that this changes would also make that property to appear in `IsInsecure` in the output of `okteto ctx show` command. Which maybe it is what we want, but just to make sure we want that